ida-partner.cer

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
+MIIDrDCCApSgAwIBAgIIE5RHue4Ke5UwDQYJKoZIhvcNAQELBQAwdjELMAkGA1UE
+BhMCSU4xCzAJBgNVBAgMAktBMRIwEAYDVQQHDAlCQU5HQUxPUkUxDTALBgNVBAoM
+BElJVEIxIDAeBgNVBAsMF01PU0lQLVRFQ0gtQ0VOVEVSIChJREEpMRUwEwYDVQQD
+DAx3d3cubW9zaXAuaW8wHhcNMjIxMjAyMTM1NDQzWhcNMjQxMjAxMTM1NDQzWjB1
+MQswCQYDVQQGEwJJTjELMAkGA1UECAwCS0ExEjAQBgNVBAcMCUJBTkdBTE9SRTEN
+MAsGA1UECgwESUlUQjEgMB4GA1UECwwXTU9TSVAtVEVDSC1DRU5URVIgKElEQSkx
+FDASBgNVBAMMC0lEQS1QQVJUTkVSMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIB
+CgKCAQEA0IoecLH6IHKZZixLNZO++hC0XtH0wUeMQ5CoFBRC6jBsyZgEHSqvGYUc
+t7r3rcLDWuiufz8VsdlzLCwV/Mw30ZB9sAut7g7wU1ak6pZfkUTLmsOgSq3szW4R
+W3u2FlMoAtUdtHClSSymJ2lwCMsKyIwfXxrB1+T0YzRISIUToYZ/atMs+ntJyyyD
+aIG1SNZi/L5CM5D0L7QhG19bgrMrgFGuk+z4CNcPBx4NHqkcowRePT3/lp98aLtB
+2Rt6FQ/naTQMRK/L19sQTvQgaYClfq9Yeuv1w449NFuJ/MixXdj3A/8OvGM6fgV7
+vHzZR8gSHNK9XpPfkyYUgXq+h2CHzQIDAQABoz8wPTAMBgNVHRMBAf8EAjAAMB0G
+A1UdDgQWBBSpx0wEBrp2hw1U8OMWzalKwaMn5DAOBgNVHQ8BAf8EBAMCBSAwDQYJ
+KoZIhvcNAQELBQADggEBAFzssGHBrBYV6OKY7PbFVge2B+SxvIRrUxFSnJIqxyGs
+mwgvEobzprH2pDir7xmQYNzG/npCw9980sJHxXRaojsby4+zxpECvTDa/00CVa3B
+4BQCumaF6aqXaY9/Eabijay9lmkKOc7robNyn0W0H39XciO0ECas+byK0EjXyvzw
+tUS2WZPnl1DXZBb6Z3yPZ/1e00dhga/IyDvBATR+++m69P0HHAOWOUKqtUDEfHTj
+kopRmdOKv70fgoXOGi0ucnCIqpJF6CiinR7PTsNXZ5IVTDQ3jdpmh9lL4Mi4gmPu
+F0SRhkXVig2LLe/OiQx7+0mjtPdKz4oEy1qeah+gCnY=
+-----END CERTIFICATE-----
```

Comment:
what is this certificate, why is this required in the config server ?

Reply:
https://mosip.atlassian.net/browse/DSD-1769 Long back @loganathan-sekaran added this will check with @balaji-alluru for more info.

Reply:
Removed file as not able to find why it's added in develop and dev-integration config.